user/src/components/Applications/StageOptions/StageOptionForm/hooks.ts

```diff
@@ -0,0 +1,158 @@
+import { useCallback, useEffect, useState } from 'react';
+import {
+  StageOptionResponse,
+  useCreateStageOptions,
+  useGetStageOptions,
+  useUpdateStageOptions,
+} from '@/api/stageOptionApi';
+import { zodResolver } from '@hookform/resolvers/zod';
+import { useForm } from 'react-hook-form';
+import { mutate } from 'swr';
+import { FormItem } from '@/components/FormList/type';
+import { stageOptionLabels } from '../../label';
+import { StageOptionForm, stageOptionSchema } from './schema';
+
+export const useStageOptionHooks = () => {
+  const {
+    handleSubmit,
+    setValue,
+    formState: { errors },
+    reset,
+    watch,
+  } = useForm<StageOptionForm>({
+    resolver: zodResolver(stageOptionSchema),
+    mode: 'onChange',
+    defaultValues: {
+      groupId: 5,
+    },
+  });
+
+  const values = watch();
+
+  const { stageOptions, isLoading, hasError } = useGetStageOptions(
+    values.groupId
+  );
+
+  const setValuesStageOptions = useCallback(
+    (res: StageOptionResponse) => {
+      setValue('ownEquipment', res.ownEquipment ? 1 : 0);
+      setValue('bgm', res.bgm ? 1 : 0);
+      setValue('cameraPermission', res.cameraPermission ? 1 : 0);
+      setValue('loudSound', res.loudSound ? 1 : 0);
+    },
+    [setValue]
+  );
+
+  useEffect(() => {
+    if (stageOptions && stageOptions !== null) {
```

Comment:
```suggestion
    if (stageOptions) {
```
stageOptionsだけでも同じかな

Reply:
{"status":{"code":200,"message":"Success"},"data":null}

nullで返ってきてたからつけたけど要らないんや！！